src/Sulu/Bundle/AdminBundle/Resources/js/components/Form/grid.scss

```diff
@@ -9,7 +9,7 @@
 .grid-section {
     margin-bottom: -$fieldMarginBottom;
 
-    & .grid-section:last-child {
+    &:last-child {
```

Comment:
What's the reasoning behind this change? Always afraid of this file, because we don't have visual regression tests until now :see_no_evil:

Reply:
if the `section` is the last child of something, we do not need to "eat" the `margin-top` of the next `item` 🙂